backend/src/main.rs

```diff
@@ -0,0 +1,55 @@
+#![allow(clippy::multiple_crate_versions)]
+
+use axum::{routing::get, Router};
+use backend::{config::ConfigExt, prelude::*, root, services::api_router};
+use cli::Parser;
+use error_stack::{Result, ResultExt};
+use std::path::PathBuf;
+use tower::ServiceBuilder;
+use tower_http::cors::CorsLayer;
+use tracing::Level;
+
+#[tokio::main]
+#[allow(clippy::unwrap_used, clippy::expect_used)]
+async fn main() -> Result<(), AppError> {
+    let config = cli::Config::try_from(cli::Args::parse())
+        .attach_printable(format!("Couldn't get config file."))
+        .change_context(AppError::InitializationError)?;
```

Comment:
Right now you attach the message to the original error, not to the `InitializationError`, but the message obviously are the description for  `InitializationError`, so you should reverse the order: first - `change_context`, second - `attach_printable`